dashboard/app/entities.go

```diff
@@ -213,6 +214,18 @@ type Commit struct {
 	Date       time.Time
 }
 
+type BugDiscussionInfo struct {
+	Source  string
+	Summary DiscussionSummary
+}
+
+type DiscussionSummary struct {
```

Comment:
The two fields are indeed present in both stucture, but I think such a refactoring won't make code better.

In `Discussion`, `Source` is a very important top-level field describing `Discussion` itself. Nesting it deeper would save one line of code in this file, but will make it look like it's just some auxiliary piece of information.

`BugDiscussionInfo` itself was only needed because datastore does not really let us encode `map`s. So I had to combine key and value using this extra structure. That's a totally different purpose.